src/components/pages/LoginHelpPage.tsx

```diff
@@ -0,0 +1,16 @@
+import parse from 'html-react-parser';
+import React from 'react';
+
+import { PageName } from 'generated/sdk';
+import { useGetPageContent } from 'hooks/admin/useGetPageContent';
+import { StyledContainer } from 'styles/StyledComponents';
+
+const LoginHelpPage: React.FC = () => {
+  const [loadingPage, pageContent] = useGetPageContent(PageName.LOGINHELPPAGE);
+
+  return (
+    <StyledContainer>{loadingPage ? null : parse(pageContent)}</StyledContainer>
```

Comment:
I think you are missing `<StyledPaper>` here after `<StyledContainer>` as that is how we keep the standard of all the pages spacings and positioning.